### ISSUE TITLE
angular CLI generating in wrong path

### ISSUE THREAD

**Issue by codex-corp** [fulls1z3/universal#450]
**I'm submitting a ...**  (check one with "x")
```
[x ] Regression (a behavior that used to work and stopped working in a new release)
[x ] Bug report  <!-- Please check the repository for a similar issue or PR before submitting -->
[ ] Support request => <!-- Please check the repository for a similar issue or PR before submitting -->
[ ] Feature request
[ ] Documentation issue or request
```

**Current behavior**

**Expected/desired behavior**

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**

**Environment**
* **Angular version:** 6.0.3

* **For Tooling issues:**
- Node version: 8 
- Platform:  Linux 

* Others:

```
hany@DESKTOP-BQMF0TU:/mnt/f/server/universal$ ng g m uikit --project client
CREATE src/client/src/app/uikit/uikit.module.spec.ts (267 bytes)
CREATE src/client/src/app/uikit/uikit.module.ts (189 bytes)
```

```
hany@DESKTOP-BQMF0TU:/mnt/f/server/universal$ ng g module todos
CREATE src/app/todos/todos.module.spec.ts (267 bytes)
CREATE src/app/todos/todos.module.ts (189 bytes)
```

On windows it does work like:

`"c:\program files\nodejs\node.exe" F:\server\universal\node_modules\@angular\cli\.\bin\ng generate component blank`

**Comment by mappedinn** [comment]
So any update for this issue???

I am facing the same issue!!!

**Comment by mappedinn** [comment]
@fulls1z3 I am proposing a PR [https://github.com/ng-seed/universal/pull/494](https://github.com/ng-seed/universal/pull/494) to solve this issue. 

@codex-corp This branch can solve your issue: [https://github.com/mappedinn/universal/tree/project-with-updated-packages](https://github.com/mappedinn/universal/tree/project-with-updated-packages)

**Comment by fulls1z3** [comment]
Solved by #507
